content/blog/2018-12-11-treestartr.md

```diff
@@ -0,0 +1,101 @@
+
+## Why ROpenSci
+
+I was a little unsure at first about submitting a package to ROpenSci. I'm a biologist, and while I'm computational, the idea that I might be a solo author on an R package seemed like I was, I don't know, misrepresenting myself? I'm not some sort of big, serious software person. But the more I thought about it, I realized that that makes ROpenSci more attractive - that some serious people could help guide me in making this package. I want stable software for my students to learn from and use. And so I should ask for review. It's appropriate and it's good to ask for expert opinion. 
+
+When I was thinking of writing up this blog, I posed the question to Twitter:
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">are there other venues (outside or within traditional scientific journals) for R code review? My packages don&#39;t fall under scope of <a href="https://twitter.com/rOpenSci?ref_src=twsrc%5Etfw">@rOpenSci</a> and would probably need domain experts (plant biologists in this case)</p>&mdash; Chris Muir (@thegoodphyte) <a href="https://twitter.com/thegoodphyte/status/1067269265810698240?ref_src=twsrc%5Etfw">November 27, 2018</a></blockquote>
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+
+And I hope I can answer this question adequately. I picked ROpenSci because it was explicitly R (so the editor and reviewers would know best practices) and the guidelines and how review would proceed were very clear. And for a totally new experience of in-depth code review, being able to see where the effort was collaborative was helpful. I looked at [other journals](https://joss.theoj.org/about#author_guidelines), but I liked that ROpenSci had biologists as editors. Because the handling editor ([Scott Chamberlain](https://scottchamberlain.info/)) is a biologist, he had the ability to work with me to find reviewers ([David Bapst](https://geogeo.tamu.edu/people/faculty/bapstdavidwilliam) and [Rachel Warnock](https://www.bsse.ethz.ch/cevo/the-group/people/person-detail.html?persid=215787)) who are serious experts in both R and the particular type of phylogenetics that I do. I chose Dave & Rachel because I know I can collaborate with them, and they will be constructive and kind. But they will also really try to break the package and dig in to the meat of what software functions are required to support the science we do, and tell me honestly if my package is providing those functions. [You can see](https://github.com/ropensci/onboarding/issues/239) they weren't easy on me!
+
+ROpenSci also has a collaboration with [Methods in Ecology and Evolution](https://besjournals.onlinelibrary.wiley.com/hub/journal/2041210X/author-guidelines). I plan to send a short software note there. It's very nice to have the flexibility to send the paper to a traditional outlet like MEE, or to send it to [JOSS](https://joss.theoj.org/), or to do nothing at all, if I didn't want to. 
```

Comment:
maybe link `[collaboration](https://ropensci.org/blog/2017/11/29/review-collaboration-mee/)`

please spell MEE the second time, rather than abbreviation